### ISSUE TITLE
Improving a Custom functions debugging documentation regarding sideloading an add-in when using Visual Studio

### ISSUE THREAD

**Issue by topkoong** [OfficeDev/office-js-docs-pr#1307]
I am writing in reference to the current documentation regarding Custon functions debugging documentation: https://docs.microsoft.com/th-th/office/dev/add-ins/excel/custom-functions-debugging. 

I was trying to use the VS Code debugger for Excel in Chrome Devtools. I know that I have to use sideloading to install an Office Add-in for testing without having to first put it in an add-in catalog, but it's kinda hard to follow. I would greatly appreciate if you guys can improve the documentation by adding some screenshots or pictures while running your add-in from VS Code as well as starting the VS Code debugger so we, developers, can follow through.

Thank you in advance.

**Comment by davidchesnut** [comment]
@keyur32 okay transferred. @topkoong thanks for this suggestion!

**Comment by mscharlock** [comment]
Just to follow up on this image request, it has been added to the list of items prepared in our image audit and will be going to design soon.

**Comment by topkoong** [comment]
I was wondering if you guys could let me know when the documentation will be released. 

Thank you!

**Comment by mscharlock** [comment]
Hi there, 
Custom functions will be undergoing some changes in the January monthly fork, which will result in some new preview documentation (which should include any changes to debugging) at the end of Jan/early Feb! Hope that helps so you have an estimation on timeline :)

**Comment by mscharlock** [comment]
Hi there @topkoong - to follow up on this, wanted to let you know about some [new documentation on our add-in debugger for VSC](https://docs.microsoft.com/office/dev/add-ins/testing/debug-with-vs-extension). Let me know if that helps. Thanks!

**Comment by mscharlock** [comment]
Hi @topkoong, I am going to close this issue for now, but let me know if you need any other assistance by commenting on this thread and I will reopen. Thank you!
